tmdb/client.py

```diff
@@ -97,6 +98,23 @@ def retrieve_show(self, show_id: int) -> Show:
         parser = self._get_show_parser()
         return parser.for_detail(data)
 
+    def get_airing_today_ids(self) -> List[int]:
+        """Retrieve IDs of shows airing today.
```

Comment:
Should we write somewhere that today is not taking into account the timezone ? I didnt check but its probably on UTC+0 timezone.

Reply:
Very good point, turns out the default timezone was UTC-5… I added a commit that uses the `TIME_ZONE` configured for the server (in `settings.py`) and set it to `Europe/Paris`. Checked and the results were indeed different than before the fix. ;-) Cheers